docs/source/deployment/knative.rst

```diff
@@ -26,16 +26,7 @@ Prerequisites
 Knative deployment with BentoML
 -------------------------------
 
-In this guide, we will build BentoService with docker and then deploy a
-prediction services on Knative with the docker image.
-
-===========================
-Build and push docker image
-===========================
-
-First, we need to build and push our BentoService to a docker registry.
-We will use the IrisClassifier BentoService from the getting
-:doc:`Quick start guide<../quickstart>`.
+This guide use the IrisClassifier BentoService from the :doc:`Quick start guide<../quickstart>`.
```

Comment:
remove the "Saved BentoService bundle" in Prerequisites, and add "pip install bentoml scikit-learn" to Prerequisites?

Reply:
can you add minikube setup link to the `Kubernetes cluster`prerequisite

Reply:
Add something like "This guide demonstrates how to deploy a machine learning model to KNative using bentoml ..." to the end of "Deploying to Knative" section at top of this file?

Reply:
actually something like what we had for kubeflow guide would be great, just modify based on this:

```
This guide demonstrates how to serve a scikit-learn based iris classifier model with BentoML on a Kubernetes cluster. The same deployment steps are also applicable for models trained with other machine learning frameworks, see more BentoML examples here.
```